quinn-udp/src/unix.rs

```diff
@@ -355,7 +369,7 @@ unsafe fn sendmmsg_fallback(
     if n == -1 {
         -1
     } else {
-        (*msgvec).msg_len = n as libc::c_uint;
+        (*msgvec).msg_len = n as _;
```

Comment:
Wouldn't mind having a comment explaining why `as _` is necessary, since it's nonobvious.

Reply:
Done: [1955853](https://github.com/quinn-rs/quinn/pull/1555/commits/195585347e37c4aa914632fdb6cd43d925d11437)